### ISSUE TITLE
API docs: http status codes 

### ISSUE THREAD

**Issue by hackebrot** [librariesio/libraries.io#1245]
Hey! 👋 

Thank you for making libraries.io 😄 

It would be great to state expected http status codes for API endpoints (see [GitHub API docs](https://developer.github.com/v3/issues/#get-a-single-issue)). This would be very useful for people, who develop API client libraries and would like to either deserialize models or return an error.

Related to #198

**Comment by andrew** [comment]
Agreed, the API documentation needs a big overhaul, this would be a great addition, plus explaining some of the potential error codes and what they mean
